azure/Guidelines.md

```diff
@@ -94,74 +94,47 @@ PUT https://blobstore.azure.com/foo.com/acct1/c1/b2?api-version=2014-12-07
 POST https://blobstore.azure.com/foo.com/acct1/c1/b2?api-version=2015-12-07
 ```
 
-### Breaking changes in Azure
+### Breaking changes
```

Comment:
Thanks Adrian for compiling this, I think it’s really great and would help a lot of us especially in Cognitive Services that iterate quite a lot on our APIs.

One tactical comment I would make is that we should consider tie the breaking changing definition with the tools that are defined in the guidelines to deal with them. The guidelines offer a SemVer versioning which includes Major.Minor. This implies that there are Major breaking changes, as well as Minor ones. Should we also say something of the form “these changes are major and these are minor”? 

I think most developers are more apprehensive about major versions upgrades than minor ones, but we have some services that are on their 8th and 12th major API versions already.

Reply:
The major thrust here is "breaking changes (except security, regulatory and API deprecation reasons) are not allowed, but if you do them, then it's a major version bump.  Everything else is a minor version bump.

Reply:
This is the gist of the thread I started. I don't know how services can evolve if adding properties have to meet that security/reg/deprecation bar

Reply:
If it is a required addition, then it is absolutely a breaking change.  An optional field with a default value would be ok.  

I think this is something that needs to be discussed during the live meeting.

Reply:
... I've just added "a new optional property is added to an existing request" as an example of a non-breaking change.  I'm not sure that's the case (round-tripping concerns), but it's a starting point on this.